test/api_integration/apis/status/status.js

```diff
@@ -34,7 +34,6 @@ export default function ({ getService }) {
           expect(body.metrics.os.cpu.load_average['5m']).to.be.a('number');
           expect(body.metrics.os.cpu.load_average['15m']).to.be.a('number');
 
-          expect(body.metrics.response_times.avg_in_millis).not.to.be(undefined); // a number, but is null if no measurements have yet been collected for averaging
```

Comment:
Why'd this get removed?

Reply:
This assertion was flaky because sometimes the field actually is undefined. If no requests have ever come to the server, this field won’t exist. Annoyingly, it gets to be null if no requests have come in during the latest five second measurement interval.

It might be better to just comment this out for now, and then add some helpers around this field in the Metrics class (`src/server/status/metrics.js`) so it is always either null or a number, never undefined.